src/common/HaskellDo/Toolbar/View.hs

```diff
@@ -40,9 +40,12 @@ toolbar = rawHtml $ do
             li ! id "packageEditorButton" $ noHtml
             li ! id "toggleEditorButton" $ noHtml
             li ! id "toggleErrorButton" $ noHtml
+            li ! id "convertToPDFButton" $ noHtml
     packageEditorModal    -- Apparently, if we put this line
     openProjectModal      -- under this one. The open project modal doesn't work
     modalPromptPlaceholder "newDirectoryModal" "New Directory" "Choose a name for the new directory"
+    convertToPDFModal
+    convertToPDFModalFail
```

Comment:
It looks like you're rendering the modals inside of the toolbar. If there's no reason for that, I'd suggest to render them in the `body` element instead, final result won't differ, but I think that makes more sense 😜